index.bs

```diff
@@ -0,0 +1,148 @@
+<pre class='metadata'>
+Title: Runtime Keys
+Shortname: runtime-keys
+Level: 1
+Status: w3c/CG-DRAFT
+Group: wintercg
+URL: http://runtime-keys.proposal.wintercg.org
+Editor: Ethan Arrowood, Vercel http://vercel.com, ethan-arrowood@vercel.com
+Abstract: A proposal defining standard identifier keys for various runtime environments
+Markup Shorthands: markdown yes
+</pre>
+<pre class=link-defaults>
+spec:url; type:interface; text:URL
+spec:html; type:attribute; for:Window; text:navigator
+spec:html; type:attribute; for:Window; text:self
+</pre>
+
+Introduction {#intro}
+=====================
+
+This proposal defines a list of keys that represent various runtime environments. These keys can be used in a variety of ways, and should be considered a reliable representation of the given runtime. For example, they may be used in a project configuration file to indicate the given project supports the specified runtime. This specification does not detail <i>how</i> the keys can be used; that is left up to community developed tools. The point of this proposal is too only define <i>what</i> the keys are in order to prevent conflicts and give users a reliable list of platforms they can build tooling around.
```

Comment:
Nit: line wrapping please :-)

Reply:
aww, soft-wrapping is so much easier to work with and has much less git diff churn. usually i add newlines after full stops tho, to aim for the best of both worlds.
```suggestion
This proposal defines a list of keys that represent various runtime environments.
These keys can be used in a variety of ways, and should be considered a reliable representation of the given runtime.
For example, they may be used in a project configuration file to indicate the given project supports the specified runtime.
This specification does not detail <i>how</i> the keys can be used; that is left up to community developed tools.
The point of this proposal is too only define <i>what</i> the keys are in order to prevent conflicts and give users a reliable list of platforms they can build tooling around.
```

Reply:
Unless there is tooling to automatically new line (i.e. prettier) I tent to not do it cause its a pain to author (as I have to constantly monitor the line length manually). Ill investigate if bikeshed has something to enforce that or see what other bikeshed specs are doing

Reply:
My suggestion isn't about length, it's just about where the sentence ends, fwiw.

Reply:
I did some digging, I found the whatwg/fetch has an `.editorconfig` file with a `max_line_length` property (https://github.com/whatwg/fetch/blob/main/.editorconfig#L10). At least for my editor of choice, VS Code, there is no support for this property. I can (personally) try to use something like Rewrap to enforce it, but I'm not seeing anything to enforce it for everyone (i.e. some linter for bikeshed).

I'm +1 for @ljharb solution.